device/macsec/mka/basic.yaml

```diff
@@ -0,0 +1,263 @@
+            aes_cmac_128:
+              x-field-uid: 1
+            aes_cmac_256:
+              x-field-uid: 2
+        key_source:
+          description: >-
+            Key source.
+          x-field-uid: 2
+          $ref: '#/components/schemas/Mka.Basic.KeySource'
+        actor_priority:
+          description: |-
+              Actor priority.
+          type: string
+          format: hex
+          minLength: 1
+          maxLength: 2
+          default: "70"
+          x-field-uid: 3
+        macsec_desired:
+          description: |-
+            Determines whether MACsec is desired or not. It is advertised in periodic Hellos.
+          type: boolean
+          default: true
+          x-field-uid: 4
+        macsec_capability:
+          description: |-
+              MACSec Capability.
+          type: string
+          default: macsec_integrity_with_confidentiality_offset
+          x-field-uid: 5
+          x-enum:
+            macsec_not_implemented:
+              x-field-uid: 1
+            macsec_integrity_without_confidentiality:
+              x-field-uid: 2
+            macsec_integrity_with_no_confidentiality_offset:
+              x-field-uid: 3
+            macsec_integrity_with_confidentiality_offset:
+              x-field-uid: 4
+        supported_cipher_suites:
+          description: |-
+            Supported Cipher Suites.
+          x-field-uid: 6
+          $ref: '#/components/schemas/Mka.Basic.SupportedCipherSuites'
+        eapol_ethernet_type:
+          description: |-
+              EAPOL Ethernet type.
+          type: string
+          format: hex
+          minLength: 1
+          maxLength: 4
+          default: "888E"     
+          x-field-uid: 7
+        mka_version:
+          description: |-
+            MKA Version.
+          type: integer
+          format: uint32
+          default: 3
+          x-field-uid: 8
+        mka_hello_time:
+          description: |-
+            MKA Hello Time (msec).
+          type: integer
+          format: uint32
+          default: 2000
+          x-field-uid: 9
+        mka_life_time:
+          description: |-
+            MKA Life Time (sec).
+          type: integer
+          format: uint32
+          default: 6
+          x-field-uid: 10
+        send_icv_indicatior_in_mkpdu:
+          description: |-
+            Send ICV Indicator in MKPDU.
+          type: boolean
+          default: true
+          x-field-uid: 11
+        delay_protect:
+          description: |-
+            Delay Protect.
+          type: boolean
+          default: true
+          x-field-uid: 12
+        rekey_mode:
+          description: >-
+            Rekey Mode.
+          x-field-uid: 13
+          $ref: '#/components/schemas/Mka.Basic.RekeyMode'
+        psk_chain_start_time:
+          description: |-
+              Pre-shared key(PSK) chain start time in UTC time format DD-MM-YYYY HH:MM:SS. The key start time will be relative to this value.
+          type: string
```

Comment:
I really do think that any string format that can be broken down into strict type checkable constituents should be done so as discussed. Otherwise user anyway has to build up the string exactly in specified format in the test program which does not seem to add much value to usability and also introduces need for additional coding for user.